home/home.js

```diff
@@ -37,4 +37,180 @@ const move = (e) => {
+      data = await response.json();
+
+    createPro(data);
+    createOffer(data.reverse());
+    createRecomeded(data.reverse())
+
+  } catch (e) {
+    console.log("error", e.message)
+  }
+})()
+
+//create dom for listOfPopulore
+function createPro(data) {
+  data.forEach(product => {
+
+    let objTitle = product.title,
+      readyTitle = objTitle.split(' ').slice(0, 3).join(' '),
+      objPrice = product.price,
+      objImage = product.image,
+      objid = product.id;
+
+    const el = document.createElement('li'),
+      proImg = document.createElement('img'),
+      textAndImg = document.createElement('div'),
+      titleAndPrice = document.createElement('div'),
+      cartImg = document.createElement('img'),
+      proTitle = document.createElement('h3'),
+      proPrice = document.createElement('span');
+
+    listOfPopulore.appendChild(el)
+    el.appendChild(proImg)
+    el.appendChild(textAndImg)
+    textAndImg.appendChild(titleAndPrice)
+    textAndImg.appendChild(cartImg)
+    titleAndPrice.appendChild(proTitle)
+    titleAndPrice.appendChild(proPrice)
+
+    el.setAttribute("class", "item")
+    proImg.setAttribute("class", "imgOfPopulore")
+    proImg.setAttribute('src', `${objImage}`)
+    textAndImg.setAttribute('class', 'wrapOfText')
+    proTitle.setAttribute('class', 'title')
+    proTitle.setAttribute('onclick', `getDataDetails(${objid},displayDetails)`)
+    proPrice.setAttribute('class', 'price')
+    cartImg.setAttribute('src', '../assets/img/cart with plus.svg')
+    cartImg.setAttribute('onclick', `getDataDetails(${objid}, addToCart)`)
+
+    proTitle.innerText = `${readyTitle}`
+    proPrice.innerText = `$${objPrice}`
+
+  });
+}
+
+//create dom for listOfOffers
+function createOffer(data) {
+  data.forEach(product => {
+
+    let objImage = product.image,
+      objCate = product.category,
+      objid = product.id;
+
+    const elemOfOffer = document.createElement('li'),
+      offerCate = document.createElement('img'),
+      offSpan = document.createElement('span'),
+      wrapOfOffer = document.createElement('div'),
+      wrapOfDesc = document.createElement('div'),
+      offImg = document.createElement('img'),
+      saveUp = document.createElement('p'),
+      disc = document.createElement('p'),
+      off = document.createElement('p');
+
+    elemOfOffer.appendChild(offerCate)
+    elemOfOffer.appendChild(offSpan)
+    elemOfOffer.appendChild(wrapOfOffer)
+    wrapOfOffer.appendChild(wrapOfDesc)
+    wrapOfOffer.appendChild(offImg)
+    wrapOfDesc.appendChild(saveUp)
+    wrapOfDesc.appendChild(disc)
+    wrapOfDesc.appendChild(off)
+    listOfOffers.appendChild(elemOfOffer)
+
+    elemOfOffer.setAttribute("class", "item")
+    elemOfOffer.setAttribute('onclick', `getDataDetails(${objid}, displayDetails)`)
+    offerCate.setAttribute('src', `../assets/img/${objCate}.svg`)
+    offerCate.setAttribute("class", "imgOfTag")
+    offSpan.setAttribute('class', 'nameOfTag')
```

Comment:
![Screenshot from 2021-08-09 09-50-30](https://user-images.githubusercontent.com/47992412/128669209-28e9dc9a-0e77-44b0-8ede-b455fece7d2d.png)

something wrong with the path, what comes from the API is diff from how you save the SVG as ... where the label renders


![Screenshot from 2021-08-09 09-52-32](https://user-images.githubusercontent.com/47992412/128669381-4811d426-6a3d-4d9a-b3e3-9107bde1a82a.png)

from the API: `men's clothing.svg` 
you save it as `men's Clothing.svg`


![Screenshot from 2021-08-09 09-54-14](https://user-images.githubusercontent.com/47992412/128669587-df099bca-2296-40db-8a75-f5c2cf6b962e.png)